sql/02-create-indexes.sql

```diff
@@ -1,35 +0,0 @@
-\connect blobber_meta;
-BEGIN;
-DROP INDEX IF EXISTS idx_unique_allocations_tx;
-CREATE UNIQUE INDEX idx_unique_allocations_tx ON allocations (tx);
-
-DROP INDEX IF EXISTS idx_pendings_cab;
```

Comment:
What does the `cab` mean, can we use meaningful name to make it easier to read, thanks.

Reply:
Previously they use client_id, allocation_id and blobber_id, shortform `cab`. Now we don't have blobber_id so I will rename it

Reply:
@peterlimg 
These all sql files are actually deleted and no more used. In the gorm tag I